src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreInstrumentationOptions.cs

```diff
@@ -52,7 +52,7 @@ public class AspNetCoreInstrumentationOptions
         /// </remarks>
         public bool RecordException { get; set; }
 
-#if NETSTANDARD2_1 || NETCOREAPP3_1 || NET5_0
+#if NETSTANDARD2_1 || NETCOREAPP3_1 || NET5_0_OR_GREATER
```

Comment:
Can be only:

```
#if NETSTANDARD2_1 || NETCOREAPP3_1_OR_GREATER
```

Reply:
The reason why we probably used `#if NETSTANDARD2_1 || NETCOREAPP3_1 || NET5_0` in the first place
https://docs.microsoft.com/en-us/dotnet/core/compatibility/sdk/5.0/netcoreapp3_1-preprocessor-symbol-not-defined

Reply:
That affects .net 5 SDK, right? So now on NET6 it should work if I undersand?

Reply:
Yes, it looks like it from the [.NET 5 and later](https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md#design) bullet here